util/fee-estimator/src/constants.rs

```diff
@@ -0,0 +1,24 @@
+//! The constants for the fee estimator.
+
+use ckb_chain_spec::consensus::{MAX_BLOCK_INTERVAL, MIN_BLOCK_INTERVAL, TX_PROPOSAL_WINDOW};
+use ckb_types::core::{BlockNumber, FeeRate};
+
+/// Average block interval.
+pub(crate) const AVG_BLOCK_INTERVAL: u64 = (MAX_BLOCK_INTERVAL + MIN_BLOCK_INTERVAL) / 2;
```

Comment:
The `AVG_BLOCK_INTERVAL` is calculated as `(48 + 8) / 2`, which equals 28 seconds. However, according to https://explorer.nervos.org/charts/block-time-distribution, most block intervals are less than 10 seconds.